subprojects/logging/src/main/java/org/gradle/internal/logging/services/LoggingServiceRegistry.java

```diff
@@ -34,36 +34,29 @@
 import org.gradle.internal.logging.source.NoOpLoggingSystem;
 import org.gradle.internal.logging.text.StyledTextOutputFactory;
 import org.gradle.internal.service.DefaultServiceRegistry;
-import org.gradle.internal.time.ReliableTimeProvider;
+import org.gradle.internal.time.MonotonicTimeProvider;
 import org.gradle.internal.time.TimeProvider;
 
 /**
  * A {@link org.gradle.internal.service.ServiceRegistry} implementation that provides the logging services. To use this:
  *
- * <ol>
- *     <li>Create an instance using one of the static factory methods below.</li>
- *     <li>Create an instance of {@link LoggingManagerInternal}.</li>
- *     <li>Configure the logging manager as appropriate.</li>
- *     <li>Start the logging manager using {@link LoggingManagerInternal#start()}.</li>
- *     <li>When finished, stop the logging manager using {@link LoggingManagerInternal#stop()}.</li>
- * </ol>
+ * <ol> <li>Create an instance using one of the static factory methods below.</li> <li>Create an instance of {@link LoggingManagerInternal}.</li> <li>Configure the logging manager as appropriate.</li>
+ * <li>Start the logging manager using {@link LoggingManagerInternal#start()}.</li> <li>When finished, stop the logging manager using {@link LoggingManagerInternal#stop()}.</li> </ol>
```

Comment:
the formatting of the javadocs is messed up. I don't think the actual content changed though

Reply:
Reverted the Javadoc formatting changes.

FWIW, this is what auto format in IDEA does for me. I cannot work out why.

Reply:
Odd. Does the same for me. It's not even consistent for methods in that class.